debezium-connector-sqlserver/src/main/java/io/debezium/connector/sqlserver/SqlServerStreamingChangeEventSource.java

```diff
@@ -165,11 +165,11 @@ public boolean executeIteration(ChangeEventSourceContext context, SqlServerParti
                             }
                         }
                         catch (SQLException e) {
-                            LOGGER.warn("No maximum LSN recorded in the database; this may happen if there are no changes recorded in the change table yet or " +
+                            LOGGER.debug("No maximum LSN recorded in the database; this may happen if there are no changes recorded in the change table yet or " +
```

Comment:
This patch should be no longer necessary. As of [CXP-2288](https://sugarcrm.atlassian.net/browse/CXP-2288) the Debezium user has permission to check the server status.